go/vt/topo/helpers/copy.go

```diff
@@ -168,8 +170,27 @@ func CopyShardReplications(ctx context.Context, fromTS, toTS *topo.Server) {
 					continue
 				}
 
+				sriNodes := sets.NewString()
```

Comment:
This `k8s.io/apimachinery` dependency seems a bit out of place. We don't usually depend on these helpers in Vitess. Since all you need is a simple String Set, you can make one with a `map[string]struct{}` that would be just as readable/ergonomic.